### ISSUE TITLE
[a11y] Service members should read hint text once

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#519]
From Terri:
Expect a screen reader user to hear the relevant hint text when radio buttons are in focus for the first time. We've been using `aria-describedby` on other pages. We should make sure it doesn't read after each option though, which is what's happening on other pages. 

hint text today is "If you’re reporting on behalf of someone else, please select their status."

**Comment by Jacklynn** [comment]
Working on VO, than you!
